.github/workflows/lint.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint
+
+on: [pull_request]
```

Comment:
This is not the appropriate config. The `pull_request` trigger by default doesn't include synchronize.

This should either be on push or have the config for `opened` and `synchronize` for the `pull_request` trigger.

Reply:
Updated in the latest commit.